docs/source/_static/custom.css

```diff
@@ -7,3 +7,20 @@ img.align-center {
     margin-bottom: 20px;
     margin-top: 20px;
 }
+
+.manim-example video {
+    width: 100%;
+}
+
+.manim-example pre {
+    border-left: none;
+    margin: 0;
+    padding: 0 0 14px 0;
+}
+
+.manim-example {
+    border-left: 2px solid #eee;
```

Comment:
Suggestion: 2px would be very small and can be increased. Also, let the border be for all sides/.

Reply:
No. This should be consistent with the current appearance of source code blocks in the documentation.